src/detect-bytemath.c

```diff
@@ -442,32 +453,27 @@ static void DetectByteMathFree(DetectEngineCtx *de_ctx, void *ptr)
  *
  * \retval A pointer to the SigMatch if found, otherwise NULL.
  */
```

Comment:
Maybe update the function description, too?

Reply:
Will do.